keps/sig-node/1797-configure-fqdn-as-hostname-for-pods/README.md

```diff
@@ -650,11 +654,13 @@ TODO
   - 99% percentile over day of absolute value from (job creation time minus expected
     job creation time) for cron job <= 10%
   - 99,9% of /health requests per day finish with 200 code
+N/A
 
 * **Are there any missing metrics that would be useful to have to improve
   observability if this feature?**
   Describe the metrics themselves and the reason they weren't added (e.g. cost,
   implementation difficulties, etc.).
+N/A
```

Comment:
L694: It doesn. For pods using this feature, the are setting the new field, which increases size of the object (by couple bytes). This is fine, but should be mentioned explicitly.

Reply:
I will add comment about increase the size of the object.